eng/TestVersions.props

```diff
@@ -1,10 +1,10 @@
 <Project>
   <PropertyGroup>
-    <NuGetFrameworksPackageVersion>5.0.0-preview1.5663</NuGetFrameworksPackageVersion>
-    <MicrosoftDotNetCliUtilsPackageVersion>3.0.100-preview.18580.6</MicrosoftDotNetCliUtilsPackageVersion>
+    <NuGetFrameworksPackageVersion>5.7.0-rtm.6702</NuGetFrameworksPackageVersion>
+    <MicrosoftDotNetCliUtilsPackageVersion>3.1.404-servicing.20561.1</MicrosoftDotNetCliUtilsPackageVersion>
```

Comment:
This doesn't look correct. I believe this should be from the 3.1.1xx band. Fixing.

Reply:
Oops, cherry-picked this from the 3.1.4xx branch, thanks for catching.